soc/st/stm32/stm32f3x/soc.c

```diff
@@ -31,6 +33,10 @@ void soc_early_init_hook(void)
 	/* At reset, system core clock is set to 8 MHz from HSI */
 	SystemCoreClock = 8000000;
 
+#if defined(CONFIG_PM) || defined(CONFIG_POWEROFF)
```

Comment:
Sorry for my previous comment.
```suggestion
#if defined(CONFIG_PM)
```
unless what build fails when PM is disabled and POWEROFF is enabled.